src/redux/actions/dataActions.js

```diff
@@ -77,13 +91,19 @@ export const dataBlockCreate = (blockData) => (dispatch) => {
 
 // update
 export const dataBlockUpdate = (blockData, blockId) => (dispatch) => {
+  console.log(blockData);
+
+  dispatch({
+    type: DATA_SET_BLOCK,
+    payload: {
+      ...blockData,
+      id: blockId,
+    },
+  });
+
   axios.patch(`/blocks/${blockId}`, blockData)
       .then((response) => {
         console.log(`[INFO] Updated: ${blockId}`);
-        dispatch({
-          type: DATA_SET_BLOCK,
-          payload: response.data,
-        });
```

Comment:
We should keep this. This way whatever the last updated value is on firebase will be used (fixes the numBursts bug)

Reply:
I tried keeping that while simultaneously doing the dispatch before the axios call and I ran into more syncing issues

Reply:
It works, but it looks a little bit jittery if going too fast however we can roll with it for now since it's synced with what is stored on firebase